config/initializers/services.rb

```diff
@@ -10,8 +10,12 @@ def self.register_service(name, service)
     @services[name] = service
   end
 
-  def self.services(name)
-    @services[name] || raise(ServiceNotRegisteredException.new(name))
+  def self.services(name = nil)
+    if name
+      @services[name] || raise(ServiceNotRegisteredException.new(name))
+    else
+      @services
+    end
```

Comment:
agreed – I've changed `SpecialistPublisher#services` to `Services#publishing_api` etc, as you suggested.

This should be ready for another look.